packages/ibm-products/src/components/Datagrid/utils/makeData.js

```diff
@@ -24,8 +24,8 @@ const getRandomInteger = (min, max, decimalPlaces) => {
 };
 
 export const makeData = (...lens) => {
-  const config = lens.filter(a => typeof a === 'object')[0];
-  const filteredData = lens.filter(a => typeof a === 'number');
+  const config = lens.filter((a) => typeof a === 'object')[0];
+  const filteredData = lens.filter((a) => typeof a === 'number');
```

Comment:
I'm not seeing any real changes. It looks like you saved this page, and it reformatted it for you.

Reply:
correct